tests/tester/tester.cpp

```diff
@@ -146,3 +146,19 @@ TEST_CASE_METHOD(eosio::test_chain, "Creating signatures", "[sign]") {
    auto r1_sig = eosio::sign(r1_pvt, hash);
    as("test"_n).act<tester_tests::assertsig_action>(hash, r1_sig, r1_pub);
 }
+
+TEST_CASE("Runtime library 128-bit integer operations") {
+   volatile int128_t a = -100, b = 2;
```

Comment:
Why do you need to declare the variable `volatile`? It only makes sense when the variable is used to communicate with a signal handler which is definitely not the case here.

Reply:
Here `volatile` keyword is to ensure compiler won’t optimize out these variables (in CMakeLists.txt, `-Os` option is used to compile `tester.cpp`); otherwise, compiler optimization will cause the int128 functions defined in `rt` library not be called.